config/routes.rb

```diff
@@ -123,7 +123,12 @@
     resource :search,    only: :show
     resource :dashboard, only: :show, constraints: { format: /html|atom/ }
     resources :profiles, only: :show
-    resource :profile, only: [:edit, :update]
+    resource :profile, only: [:edit, :update] do
+      member do
+        get :delete
+        delete :destroy, as: :destroy
```

Comment:
If we change
```Ruby
resources :profiles, only: :show
```
to
```Ruby
resources :profiles, only: [:show, :destroy]
```
We have a path `DELETE	/profiles/:id(.:format)` which not really the format rest of pages follow(for ex we have `/profile`, `/profile/edit`)